src/app/ui-kit/button/button.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-button',
```

Comment:
let's add ui or ui-kit in selector name to mark out ui-kit components (for example, app-ui-button)